crypto/x509/standard_exts.h

```diff
@@ -13,14 +13,6 @@
  */
 
 static const X509V3_EXT_METHOD *standard_exts[] = {
-    &v3_nscert,
-    &v3_ns_ia5_list[0],
```

Comment:
`v3_ia5.c` file should be removed too, as nothing more is using `v3_ns_ia5_list`

Reply:
ooh, cool.  removed the file.